tests/Copy-DbaTableData.Tests.ps1

```diff
@@ -5,15 +5,18 @@ Write-Host -Object "Running $PSCommandpath" -ForegroundColor Cyan
 Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
     BeforeAll {
         $db = Get-DbaDatabase -SqlInstance $script:instance1 -Database tempdb
+        $db2 = Get-DbaDatabase -SqlInstance $script:instance2 -Database tempdb
```

Comment:
This is not allowed so will have to be removed. We cannot connect between two VMs in Appveyor environment. As well, I would think even doing it would cause instability in our testing process.

Reply:
If you want to add a test for the parameter you fixed simply create another database on the same Appveyor instance. Then that would provide code coverage for the parameter you fixed.

Reply:
Ok...I have that wrong this is apparently allowed.

Reply:
Sorry, my bad. I misunderstood! We allow instance to instance but not VM to VM.